### DISCUSSION TITLE
Is awscrt OS dependent ?

### DISCUSSION THREAD

**Discussion by Neenu-12** [awslabs/aws-crt-python#551] [Q&A] [answered]
As there are files written in C for awscrt and of course these files will be compiled differently depending on the platform why awscrt is still classified as OS independent ? 
Please refer to the below link referring to setup.py :-
https://github.com/awslabs/aws-crt-python/blob/main/setup.py#L414
Also is mentioned as os-independent on PyPI website :-
https://pypi.org/project/awscrt/

I was working on a project few days back that required awscrt , at first site it seemed to be OS independent as I had a look at the classifier but didn't work as expected because I downloaded and zipped awscrt of a windows machine to add it in the layer for a lambda function.

Either I'm missing something or classifier needs to be fixed so that it doesn't cause confusion.

**Comment by waahm7** [comment, marked as answer]
Thank you for creating the issue. I have created a [PR](https://github.com/awslabs/aws-crt-python/pull/552) to fix the classifier. `awscrt` contains native code and platform-specific wheels. You will need to zip the target platform instead of the host platform, as recommended [here](https://docs.aws.amazon.com/lambda/latest/dg/python-package.html#python-package-native-libraries) for native libraries.
